src/common/components/vanilla/heading/index.js

```diff
@@ -4,15 +4,17 @@ import styles from './heading.css';
 
 const Heading = (props) => {
   const H = props.heading || 'h1';
+  const A = props.align || 'left';
   return (
-    <H className={ styles[H] }>
+    <H className={ `${styles[H]} ${styles[A]} `}>
       { props.children }
     </H>
   );
 };
 
 Heading.propTypes = {
   children: PropTypes.node,
+  align: React.PropTypes.oneOf(['left', 'right', 'center']),
```

Comment:
Maybe it doesn't matter so much ATM, but there are no such variants of headings in vanilla.

In theory, if we want this heading component to be equivalent of vanilla it should only have styles of vanilla headings.